src/elements/content-sidebar/additional-tabs/AdditionalTabs.js

```diff
@@ -27,40 +27,22 @@ class AdditionalTabs extends PureComponent<Props, State> {
         this.state = { isLoading: true };
     }
 
-    componentDidMount() {
-        this.calculateNumberOfLoadableTabs();
-    }
-
-    componentDidUpdate() {
-        this.calculateNumberOfLoadableTabs();
-    }
-
     /**
-     * Determines if we need to account for a "more" tab when loading icon images.
+     * Handles an individual icon image load
      *
      * @return {void}
      */
-    calculateNumberOfLoadableTabs() {
+    onImageLoad = () => {
         const { tabs } = this.props;
 
-        if (tabs) {
-            // If we're displaying the more options tab, consider it loaded since it doesn't use an image.
-            const hasMoreTab = tabs.find(tab => tab.id < 0 && !tab.iconUrl);
-            const moreTabCount = hasMoreTab ? 1 : 0;
-
-            this.numLoadedTabs = moreTabCount;
+        if (!tabs) {
+            return;
         }
-    }
 
-    /**
-     * Handles individual icon image load
-     *
-     * @return {void}
-     */
-    onImageLoad = () => {
-        const { tabs } = this.props;
+        let numTabs = tabs.length;
+        const hasMoreTab = tabs.find(tab => tab.id < 0 && !tab.iconUrl);
+        numTabs -= hasMoreTab ? 1 : 0;
```

Comment:
General convention is to put `let` after `const`, which would also remove the need for reassignment.